tools/fog-local-network/README.md

```diff
@@ -58,7 +58,8 @@ In order to use it, the following steps are necessary.
     # Create a set of target keys. They would be identical to the first N keys inside `keys/`. This is needed if you don't
     # want to send to transactions to all 1000 keys created at step 1.
     # Notice the addition of the --output-dir argument
-    cargo run -p mc-util-keyfile --bin sample-keys --release -- --num 10 --output-dir fog_keys --fog-report-url 'insecure-fog://localhost:6200' --fog-authority-root $(../target/release/mc-crypto-x509-test-vectors --type=chain --test-name=ok_rsa_head)
+    cargo run -p mc-util-keyfile --bin sample-keys --release -- --num 10
+    --output-dir fog_keys --fog-report-url 'insecure-fog://localhost:6200' --fog-authority-root $(${CARGO_TARGET_DIR:-target}/release/mc-crypto-x509-test-vectors --type=chain --test-name=ok_rsa_head)
```

Comment:
Remove the line break, or add `\` before it

```suggestion
    AUTHORITY_ROOT=$(${CARGO_TARGET_DIR:-target}/release/mc-crypto-x509-test-vectors --type=chain --test-name=ok_rsa_head)
    cargo run -p mc-util-keyfile --bin sample-keys --release --
        --num 10 \
        --output-dir fog_keys \
        --fog-report-url 'insecure-fog://localhost:6200' \
        --fog-authority-root $AUTHORITY_ROOT
```

Reply:
pushed up a resolution in newest commit